src/apps/material/material.test.ts

```diff
@@ -129,12 +178,27 @@ describe("Material", () => {
     );
     cy.get("#year").select("2021");
     cy.get("#editions").should("have.value", "52");
-    cy.contains("button:visible", "Reserve periodikum").click();
-    cy.contains("h2", "2021, nr. 52");
-    cy.contains("button:visible", "Approve reservation").click();
-    cy.contains("Material is available and reserved for you!");
-    cy.contains("You are number 3 in the queue");
-    cy.contains("button:visible", "Ok").click();
+    cy.getBySel("material-header-buttons-physical")
+      .should("be.visible")
+      .and("contain", "Reserve periodikum")
+      .click();
+
+    cy.get("h2").should("contain", "2021, nr. 52");
```

Comment:
Should this h2 have a cypress attribute?

Reply:
I thought in this case that there is only one h2 in the DOM so I did not want to use time on it.